index.js

```diff
@@ -7,12 +18,95 @@ function ask(questionText) {
   });
 }
 
-start();
+// starts the pgm - 
+start();   
 
+// function starting the pgm
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  startGameSetup();
+  // hard coding to min & max to begin with 
+    let min=1;    
+    let max=100;  
+    range = ((max - min)+1);  
+
+    console.log("Please think of a number between " + min + " and " + max + " (inclusive), in this " + range + " number range.");
+    let secretNumber = await ask("What is your secret number?\nThis Javascript (JS) program (pgm) won't peek...\n");
```

Comment:
You don't have to ask the user for a secret number. They're already thinking about that number as they are playing the game

Reply:
I left that code in there from the original I was given; I think it should remain; I use it and will use it to finish the code to compare final answer with how they answer (also for cheat part of code).